test/system/departments_test.rb

```diff
@@ -62,7 +62,7 @@ def setup
     assert page.has_text? 'department@aoz.ch'
     assert page.has_text? '0441234567'
     assocable_users.each do |user|
-      assert page.has_link? user.to_label
+      assert page.has_link? user.full_name, href: profile_path(user.profile)
```

Comment:
Would this also work if you did this like that?
```ruby
within '.container` do
  assert page.has_link? user.full_name
end
```
This way you'd not need to pass href